pymeasure/adapters/prologix.py

```diff
@@ -107,15 +110,66 @@ def __init__(self, resource_name, address=None, rw_delay=0, serial_timeout=None,
                          **kwargs)
         self.address = address
         if not isinstance(resource_name, PrologixAdapter):
-            self.set_defaults()
+            self.auto = auto
+            self.eoi = eoi
+            self.eos = eos
+
+    @property
+    def auto(self):
+        """Configure Prologix GPIB controller to automatically address instruments
+        to talk after sending them a command in order to read their response. The
+        feature called, Read-After-Write, saves the user from having to issue read commands
+        repeatedly. This property enabled or disabled the Read-After-Write feature.
+        """
+        self.write("++auto")
+        return int(self.read())
+
+    @auto.setter
+    def auto(self, value):
+        self.write("++auto {}".format(value))
+
+    @property
+    def eoi(self):
+        """Enables or Disables the assertion of the EOI signal with the last character
+        of any command sent over GPIB port. Some instruments require EOI signal to be
+        asserted in order to properly detect the end of a command.
+        """
+        self.write("++eoi")
+        return int(self.read())
+
+    @eoi.setter
+    def eoi(self, value):
+        self.write("++eoi {}".format(value))
+
+    @property
+    def eos(self):
+        """Specify GPIB termination characters. When data from host is received,
+        all non-escaped LF, CR and ESC characters are removed and GPIB
+        terminators, as specified by this command, are appended before sending the data to
+        instruments. This command does not affect data from instruments received over GPIB
+        port.
```

Comment:
I think also valid inputs values and their meaning should be specified in the doc string.